code/src/App.js

```diff
@@ -1,12 +1,22 @@
 import React from 'react'
+import { Header } from './components/Header'
+import { Album } from './components/Album'
 import data from './data.json'
+import './app.css'
 
-console.log(data)
+const albumArray = data.albums.items;
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <>
+      <Header headerText="New albums &amp; singles"/>
+      <section className="album-list">        
+
+        {/* Mappar igenom allt under items, anropar Album.js för varje album i data.json. Skickar med props. */}
```

Comment:
Keep your code (including comments and variable names) to English so that everyone can understand your code :)

Reply:
Fixed